apk-generator/index.html

```diff
@@ -121,13 +121,19 @@ <h3 id="status"></h3>
                 $('#generator-progress').css('display', 'block')
                 $('#upload-progress').css('display', 'block')
                 var ary = $(form).serializeArray();
+                var emailse = $("#Email").val();
                 var obj = {};
                 updatePercentUpload(10);
                 for (var a = 0; a < ary.length; a++) obj[ary[a].name] = ary[a].value;
                 console.log("JSON", obj);
                 if (obj.Email == "" || obj.App_Name == "") {
```

Comment:
@the-dagger, sure but that would decrease the user experience. For example think of you having to fill up 20 forms with various attitudes[email,name,phone number etc] and you filled up wrong and then it shows that you entered no details at all! I agree that it can increase speed but only a bit and that bit is useless if you give up a lot of user experience the other end. We have to code for the user and not for us or anything that would look good for a programmer! :)